sql/migrations/20191208135917_world.sql

```diff
@@ -0,0 +1,38 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20191208135917');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20191208135917');
+-- Add your query below.
+
+
+DELETE FROM `script_texts` WHERE `entry` IN (-1000715, -1000716, -1000717, -1000718, -1000719, -1000720, -1000721, -1000722, -1000723, -1000724, -1000725, -1000726, -1000727, -1000728, -1000729);
+DELETE FROM `script_texts` WHERE `entry` IN (-1000730, -1000731, -1000732, -1000733, -1000734, -1000752, -1000735, -1000736, -1000737, -1000738, -1000739, -1000740, -1000741, -1000742, -1000743);
+DELETE FROM `script_texts` WHERE `entry` IN (-1000744, -1000745, -1000746, -1000747, -1000748, -1000749, -1000750, -1000751, -1000672, -1000673, -1000674, -1000676, -1000678, -1000680, -1000683);
+DELETE FROM `script_texts` WHERE `entry` IN (-1000684, -1000675, -1000677, -1000679, -1000681, -1000682, -1000323, -1000324, -1780189, -1000106, -1000107, -1000108, -1000109, -1000110, -1000817);
+DELETE FROM `script_texts` WHERE `entry` IN (-1000818, -1000819, -1000820, -1000821, -1000822, -1000823, -1000824, -1000825, -1000826, -1780210, -1000800, -1000434, -1000435, -1000436, -1000437);
+DELETE FROM `script_texts` WHERE `entry` IN (-1000438, -1000439, -1000440, -1000441, -1000445, -1000442, -1000443, -1000444, -1000327, -1000339, -1000328, -1000329, -1000330, -1000331, -1000332);
+DELETE FROM `script_texts` WHERE `entry` IN (-1000333, -1000334, -1000335, -1000336, -1000337, -1000338, -1000325, -1000326, -1000905, -1000906, -1000789, -1000790, -1000791, -1000792, -1000793);
+DELETE FROM `script_texts` WHERE `entry` IN (-1000794, -1000795, -1000796, -1780209, -1780226, -1780184, -1780185, -1780186, -1780187, -1780188);
+
+UPDATE `broadcast_text` SET `EmoteId0` = 1 WHERE `ID` IN (11293, 11295, 11301, 11302, 1244, 1236);
+UPDATE `broadcast_text` SET `EmoteId0` = 2 WHERE `ID` IN (8265);
+UPDATE `broadcast_text` SET `Type` = 1 WHERE `ID` IN (11030, 11293, 11294, 11295, 11297, 11299, 11304, 11305, 11306, 11309, 11310, 11311, 11314, 11315, 11316, 11320);
+UPDATE `broadcast_text` SET `Type` = 2 WHERE `ID` IN (11313, 11160, 3012);
+UPDATE `broadcast_text` SET `Type` = 3 WHERE `ID` IN (11277);
+UPDATE `broadcast_text` SET `Language` = 1 WHERE `ID` IN (8284, 8278, 8282, 8280, 8281);
+
+UPDATE `broadcast_text` SET `MaleText` = "A $c! Slaying him would please the master. Attack!" AND `FemaleText` = "A $c! Slaying him would please the master. Attack!" WHERE `ID` = 1309;
+UPDATE `broadcast_text` SET `MaleText` = "%s, has entered our world." AND `FemaleText` = "%s, has entered our world." WHERE `ID` = 11277;
+UPDATE `broadcast_text` SET `MaleText` = "%s, is wholly consumed by the Light of Elune. Tranquility sets in over the Moonglade." AND `FemaleText` = "%s, is wholly consumed by the Light of Elune. Tranquility sets in over the Moonglade." WHERE `ID` = 11313;
```

Comment:
Do not modify the text columns of broadcast_text table! The texts are all 100% correct as they are sniffed from the official Classic servers.

Reply:
Very strange, that expression of behalf of here is used not through '%s'. Well. I undoed the changes.